content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/showhidedialogfields/source/showhidedialogfields.js

```diff
@@ -82,6 +82,10 @@
     	if(select){
     		return "select";
     	}
+	    
+	if($(element).hasClass("coral-Checkbox")){
```

Comment:
(commented on wrong PR)
Per the comment at the top of https://helpx.adobe.com/experience-manager/6-3/sites/developing/using/reference-materials/coral-ui/coralui3/Coral.Checkbox.html, this class shouldn't be referenced directly. I believe the correct check would be on the element name.

Reply:
@justinedelson:

Reply:
According to https://stackoverflow.com/questions/8852665/how-can-i-get-name-of-element-with-jquery, you should use attr

Reply:
But "./frame" has nothing to do with checkbox. I don't think we can identify a checkbox with this in a reliable way

Reply:
Sorry, I edited my comment above to clarify that you should look at the element name, not the name property (or attribute). Strictly speaking, in DOM terms this is "tag name".

Reply:
I changed it to use tag name